test/hotspot/jtreg/runtime/cds/appcds/ClassPathAttr.java

```diff
@@ -82,6 +83,40 @@ static void testNormalOps() throws Exception {
             output.shouldMatch("checking shared classpath entry: .*cpattr2.jar");
             output.shouldMatch("checking shared classpath entry: .*cpattr3.jar");
           });
+
+      // Test handling of forward slash ('/') file separator for Class-Path attribute on Windows.
+      if (Platform.isWindows()) {
```

Comment:
Does the new test work for other platforms as well? If so, I think we should remove this check.

Reply:
It does. Since other platforms have '/' as the file separator, running it again would be wasting of time.

Reply:
That makes sense.